genie-common-internal/src/main/java/com/netflix/genie/common/internal/configs/CommonServicesAutoConfiguration.java

```diff
@@ -71,47 +66,27 @@ public FileSystemJobArchiverImpl fileSystemJobArchiver() {
      * Provide a default {@link JobArchiveService} if no override is defined.
      *
      * @param jobArchivers                The ordered available {@link JobArchiver} implementations in the system
-     * @param jobDirectoryManifestService the job directory manifest service
+     * @param directoryManifestFactory the job directory manifest factory
      * @return A {@link JobArchiveServiceImpl} instance
      */
     @Bean
     @ConditionalOnMissingBean(JobArchiveService.class)
     public JobArchiveService jobArchiveService(
         final List<JobArchiver> jobArchivers,
-        @Qualifier(JOB_DIRECTORY_MANIFEST_SERVICE_CS_BEAN) final JobDirectoryManifestService jobDirectoryManifestService
-    ) {
-        return new JobArchiveServiceImpl(jobArchivers, jobDirectoryManifestService);
-    }
-
-    /**
-     * Provide a {@link JobDirectoryManifestService} if no override is defined.
-     * The manifest produced by this service include checksum for each entry.
-     *
-     * @param directoryManifestFactory the factory to produce the manifest if needed
-     * @return a {@link JobDirectoryManifestService}
-     */
-    @Bean(name = JOB_DIRECTORY_MANIFEST_SERVICE_CS_BEAN)
-    @ConditionalOnMissingBean(
-        name = JOB_DIRECTORY_MANIFEST_SERVICE_CS_BEAN
-    )
-    public JobDirectoryManifestService checksummingJobDirectoryManifestService(
         final DirectoryManifest.Factory directoryManifestFactory
     ) {
-        return jobDirectoryPath -> directoryManifestFactory.getDirectoryManifest(jobDirectoryPath, true);
+        return new JobArchiveServiceImpl(jobArchivers, directoryManifestFactory);
     }
 
     /**
      * Provide a {@link JobDirectoryManifestService} if no override is defined.
-     * The manifest produced by this service do not include checksum for each entry.
+     * The manifest produced by this service do not include checksum for entries and caches manifests recently created.
```

Comment:
I might be missing something but there doesn't seem to be anything here to indicate that it caches other than your comment

Reply:
I'm instantiating an `ServiceImpl` that does cache.